Cargo Reworks

CargoVroom opened the pull request:
Reworks of FCS, Actuators as discussed. Make arguments for changing various values from here pls since it seems like the concept is generally accepted.

MRM range nerf to canonical value. Notably affects MMS. (Oddly enough BA MRM were actually under-range)
Rocket ranged nerfed to canonical, they were even longer ranged than MRM.

Small soft buff to pulse weapons. +1 Acc +1 Pip to +2 Acc. This will be one of the more noteworthy acc buffs in the game now as it should be.

Ongoing minor reworks and corrections to BA weapons related to my MG rework, and testing.

Changed/Corrected various lasers to canonical values as a basis for future balancing. Some of our values, like Clan ER Small lasers which we both buffed and nerfed pushed them out of their concept IMO.

Limiting relative height accuracy bonus/malus to +/- 1, following discussions with Edraii about overwhelming modifiers on some maps and cheese potential.

CargoVroom committed:
Nerf +Accuracy, +Range, -Heat

Some of this is OP. +3 Gauss accuracy? -15% Laser heat? Really? Most of this is bloat. or keeping up with power creep. If this much is nessasary for stuff to be viable we have better more global ways of manipulating global hit odds

CargoVroom committed:
Nerf +Accuracy, +Range, -Heat

Some of this is OP. +3 Gauss accuracy? -15% Laser heat? Really? Most of this is bloat. or keeping up with power creep. If this much is nessasary for stuff to be viable we have better more global ways of manipulating global hit odds

CargoVroom committed:
Cavalry VTOl Resize

It uses the same model as the Warrior, and was set slightly smaller, despite weighing slightly more.

CargoVroom committed:
MRM/rocket range fixes

BA MRM were too short, normal MRM were too long. Weird. Now all MRM and MRM Derived systems are canonical 90/240/450/480

All Rocket ranges reduced to canonical 90/210/360/420

CargoVroom committed:
BA Weapon tweaks and fixes

CargoVroom committed:
Update BonusDescriptions_AmmoTypes.json

CargoVroom committed:
Update Weapon_Plasma_Rifle.json

CargoVroom committed:
Double Bins to +10%

Large Chem to 12 base from 16.
RAC2 base to 45 from 60.

CargoVroom committed:
Update Gear_QuadVee_Turret.json

CargoVroom committed:
Pulse acc tweak

From +1 Acc +1 Pip to canonical +2 Accuracy. This is a soft buff. (Additional bonus acc removed from IS pulse variants)

CargoVroom committed:
Merge branch 'Dev' into cargo-temp

CargoVroom committed:
oops

CargoVroom committed:
micro laser range fixes

CargoVroom committed:
Update CombatGameConstants.json

Removing stacking elevation buff/malus per discussions with Edraii about extreme gains on some maps. Max change is now +/- 1

CargoVroom committed:
HAG40 fix

Corrected erroneous +3 vs VTOL to +2.
RISC HAG corrected bonus description to reflect the +2 it actually has.

CargoVroom committed:
Weapon heat and range fixes/tweaks

knan commented on Core/RogueModuleTech/Cockpit/FCS/Gear_FCS_Ballistic_Accuracy.json:
Updated description but forgot to update effect to match

CargoVroom replied:
Fixing, thanks

knan commented on Core/RogueModuleTech/Cockpit/FCS/Gear_FCS_Ballistic_Improved.json:
Here too, updated descriptions but didn't change effects

CargoVroom replied:
Fixing, thanks

knan commented on Core/RogueModuleTech/Cockpit/FCS/Gear_FCS_Energy_Crit.json:
Updated descriptions but not effects

CargoVroom replied:
Fixing, thanks

CargoVroom committed:
Update Weapon_BattleArmor_Needler.json

CargoVroom committed:
Update Weapon_Flamer_ER.json

CargoVroom committed:
Evasion Ignore [Location]

Hope this works.

CargoVroom committed:
Update Weapon_Autocannon_RAC_2_Clan.json

CargoVroom committed:
FCS Fixes

CargoVroom committed:
LRM descriptions

Past me removing these labels was silly.

CargoVroom committed:
MG Fixes

CargoVroom committed:
FCS mg

CargoVroom committed:
Turret temp(?) rebalance

I know other things are under consideration but for now in keeping with the FCS and actuators this is too high a bonus.

CargoVroom committed:
Pulse Module again

Because I got one to play with and it was staggeringly worse than it's marginally usable tabletop original.

CargoVroom committed:
Merge branch 'cargo-temp' of https://github.com/BattletechModders/RogueTech into cargo-temp

CargoVroom committed:
weird mg ranges

updated to match rework.

CargoVroom committed:
Missile and Ballistic heat fixes

It turns out our perhaps overdominant ballistics and missiles almost universally ran slightly cooler than on TT. So just undoing that seems like a good step rather than buffing a lot of energy weapon heat. Energy running 'too hot' is a relative issue as much as an absolute one.

CargoVroom committed:
SM capped at 4

as part of general reduction in modifier bloat.
